### PR TITLE
fix(styles): add basic color-scheme support

### LINKED ISSUE #2650: [Bug]: On iOS devices using Dark theme, the scrollbar color for sp-menu component has a very light color that is hard to see
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Menu

### Expected behavior

On iOS devices using Dark theme, the scrollbar color for `sp-menu` component has a very light color that is hard to see for user. The color needs to be a bit darker for usability

### Actual behavior

Color is very light makes it almost invisible.

### Screenshots

iOS Dark Mode
iOS-DarkMode

iOS Light Mode
iOS-LightMode

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. On an iOS device Change your appearance to Dark: Settings > Display & Brightness > Dark
2. Go to [this example](https://studio.webcomponents.dev/edit/mdfS0dxsh3GuS6wsbj0B/src/index.ts?sv=1&pm=1)
3. Toggle the iphone view 
PixelSnap 2022-10-10 at 14 47 37

4. Scroll to the menu list
5. Observe the scrollbar color is very light 

### Sample code that illustrates the problem

https://studio.webcomponents.dev/edit/mdfS0dxsh3GuS6wsbj0B/src/index.ts?pm=1&p=stories

### Logs taken while reproducing problem

_No response_

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
Deliver browser chrome within an theme root in the theme that is described by that root.

## Related issue(s)
- fixes #2650

## Screenshots (if appropriate)
Before:
image

After:
image

## Types of changes
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist
-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

### CODE REVIEW

File: projects/documentation/src/components/styles.css [hunk 1]
```diff
@@ -59,7 +59,6 @@ governing permissions and limitations under the License.
 
 body {
     margin: 0;
-    background-color: var(--spectrum-global-color-gray-50);
```
Comment (Westbrook) [inline projects/documentation/src/components/styles.css:62]: Allows the browser to decide the far background color based on the `color-scheme`.

File: projects/documentation/src/components/layout.ts [hunk 2]
```diff
@@ -454,6 +454,12 @@ export class LayoutElement extends LitElement {
                         ? 'dark'
                         : 'light';
             });
+            (
+                document.querySelector('html') as HTMLHtmlElement
```
Comment (Westbrook) [inline projects/documentation/src/components/layout.ts:458]: Pushes the `color-scheme` outside of the app surface.

File: scripts/spectrum-vars.js [hunk 3]
```diff
@@ -230,7 +230,7 @@ spectrumPaths.forEach(async (spectrumPath, i) => {
                 '..',
                 'tools',
                 ...packageDir,
-                `theme-${theme}.css`
+                `spectrum-theme-${theme}.css`
```
Comment (Westbrook) [inline scripts/spectrum-vars.js:233]: Create the Spectrum specific version of this file and then point to it with our entry point where we've added the `color-scheme` information.

**Review by najikahalsema — APPROVED** [review]
Nice!

**Pull request merged**